kernel/src/sched/priority.rs

```diff
@@ -62,14 +62,18 @@ impl TryFrom<i8> for Nice {
 #[derive(Clone, Copy, Debug, Eq, PartialEq, PartialOrd, Ord)]
 pub struct Priority(PriorityRange);
 
-pub type PriorityRange = RangedU8<0, 139>;
+pub type PriorityRange = RangedI16<-1, 140>;
```

Comment:
I am wondering if `Priority` is of any real value given the fact that the same information is also encoded inside `SchedPolicy`. So to me, `Priority` will soon be discarded after we completely transit to the new class-based scheduler. And thus, enlarging its value range is meaningless.